test/jdk/java/lang/RuntimeTests/RuntimeExitLogTest.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2023, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2025, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Hello @tkiriyama, this should be `2023, 2025, `.

Reply:
I'm sorry. I fixed this mistake.